tests/unit/test_oidc.py

```diff
@@ -0,0 +1,226 @@
+"""
+Unit tests for the oidc code in the frontend.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import ga4gh.frontend as frontend
+import tests.utils as utils
+from mock import patch
+import urlparse
+import oic
+import oic.oic.message as message
+
+RANDSTR = 'abc'
+OICCODE = "5GzAsW4Gfz8Km8PTuayFi6j2tA2VnPrHiWybL%2BkryEpr05D0OUKTGTi49Z54yl" \
+          "%2FIB5KsUzGwSg%2BFpx%2FpBt0aVc3QsvPlsJu6WGHIchJ55UU%3D"
+
+OICRESPONSE = {'scope': u'openid profile',
```

Comment:
I tend to prefer hanging indents over vertical alignment, e.g.,
```python
OICRESPONSE = {
    'scope': u'openid profile', 
    'state': RANDSTR,
    'code': (
        u'5GzAsW4Gfz8Km8PTuayFi6j2tA2VnPrHiWybL+kryEpr05D0OUKTGTi49Z54yl/IB5KsUzGwSg+'
        u'px/pBt0aVc3QsvPlsJu6WGHIchJ55UU='
    )
}
```
This is a very subjective style issue though, so I won't push hard on this. I tend to prefer hanging indents because they are more robust to code changes. For example, when we change method names with a vertical alignment, we need to realign all the arguments. With a hanging indent this isn't necessary. I agree this is very subjective though, so this is just a comment rather than a request to change anything.